File: Day9_Polymorphism-1/2_Shape/Square.cpp
#include"Square.h"

Square::Square()
{
    side=5;
}
Square::Square(int s)
{
    side=s;
}
long Square::calculateArea()
{
    cout<<"Area of Square: ";
    area=(side*side); 
    return area;
}
long Square::calculatePerimeter()
{
    cout<<"Perimeter of Square: ";
    perimeter=4*side;
    return perimeter;
}

File: Core_java_new/_09Interface/A1_Shapes/shapes/Square.java
package shapes;

import shapeutil.Drawable;

public class Square implements Drawable {

	private int side;
	
	public Square() {////////////////////////////////////////
		super();
	}
	public Square(int side) {
		super();
		this.side = side;
	}


	@Override
	public void drawShape() {///////////////////////////////
		System.out.println("In drawShpae() of Square");
	}

	@Override
	public void calArea() {
		System.out.println("Area of Square: "+ side*side);
	}

}
